### PR TITLE
Update z method to standardize

### PR DESCRIPTION
**Pull request by 0xMuluh**
This commit updates the z method to `standardize`.
ping: [Comment](https://github.com/microbiome/mia/pull/584#issuecomment-2186283351)

### CODE REVIEW

File: inst/pages/11_taxonomic_information.qmd [hunk 1]
```diff
@@ -336,10 +336,10 @@ references.
    in microbiome-based classification performance [@Giliberti2022,
    Karwowska2024].
 
- * 'z' Z transformation scales data to zero mean and unit variance;
-   this is used to bring features (or samples) to more comparable
-   levels in terms of mean and scale of the values. This can enhance
-   visualization and interpretation of the data
+ * 'standardize' standardize transformation scales data to zero mean
```
Comment (TuomasBorman) [inline inst/pages/11_taxonomic_information.qmd:339]: You can add that it is also known as Z-transformstion

File: inst/pages/add-comm-typing.Rmd [hunk 2]
```diff
@@ -502,19 +502,19 @@ bray_dmm_plot
 Visualise dmm clusters in a heatmap.
 
 ```{r}
-# get clr + z-transformed counts
+# get clr + standardize-transformed counts
```
Comment (TuomasBorman) [inline inst/pages/add-comm-typing.Rmd]: This sentence sounds odd

File: inst/pages/add-comm-typing.Rmd [hunk 3]
```diff
@@ -337,18 +337,18 @@ scater::plotReducedDim(tse2, "NMDS_BC", colour_by = "CST", point_alpha = 1) + th
...
 # Z transform of CLR counts
 tse2 <- transformAssay(tse2, MARGIN = "samples", assay.type = "counts",
                         method = "clr", pseudocount=1)
-tse2 <- transformAssay(tse2, MARGIN = "features", assay.type = "clr", method = "z")
+tse2 <- transformAssay(tse2, MARGIN = "features", assay.type = "clr", method = "standardize")
 # get top taxa
 tse2 <- tse2[is.element(rownames(tse2), rel_taxa), ]
 
-mat <- assays(tse2)$z
+mat <- assays(tse2)$standardize
```
Comment (TuomasBorman) [inline inst/pages/add-comm-typing.Rmd]: Use assay(tse2, "standardize")

File: inst/pages/add-comm-typing.Rmd [hunk 4]
```diff
@@ -168,15 +168,15 @@ tse <- mergeFeaturesByRank(tse, rank = "Phylum", agglomerateTree=TRUE)
...
 tse <- transformAssay(tse, MARGIN = "samples", assay.type = "counts",
                        method = "clr", pseudocount=1)
-tse <- transformAssay(tse, MARGIN = "features", assay.type = "clr", method = "z")
+tse <- transformAssay(tse, MARGIN = "features", assay.type = "clr", method = "standardize")
 
 Countries <- data.frame("Country" = colData(tse)$Country)
 rownames(Countries) <- colData(tse)$Sample_ID
 # count matrix for pheatmap
-mat <- assays(tse)$z
+mat <- assays(tse)$standardize
```
Comment (TuomasBorman) [inline inst/pages/add-comm-typing.Rmd]: Use assay(tse2, "standardize")

File: inst/pages/add-comm-typing.Rmd [hunk 5]
```diff
@@ -502,19 +502,19 @@ bray_dmm_plot
...
-                           assay.type = "clr", method = "z")
+                           assay.type = "clr", method = "standardize")
 # objects containing dmm component information
 clust <- factor(vec)
 names(clust) <- colnames(tse_dmn)
 # get top taxa
 tse_dmn <- tse_dmn[is.element(rownames(tse_dmn), rel_taxa), ]
 # get just counts
-mat <- assays(tse_dmn)$z
+mat <- assays(tse_dmn)$standardize
```
Comment (TuomasBorman) [inline inst/pages/add-comm-typing.Rmd]: Use assay(tse2, "standardize")

File: inst/pages/11_taxonomic_information.qmd [hunk 6]
```diff
@@ -336,10 +336,11 @@ references.
...
 
- * 'z' Z transformation scales data to zero mean and unit variance;
-   this is used to bring features (or samples) to more comparable
-   levels in terms of mean and scale of the values. This can enhance
-   visualization and interpretation of the data
+ * 'standardize' standardize transformation also known as Z-transformation
+   scales data to zero mean and unit variance; this is used to bring 
+   features (or samples) to more comparable levels in terms of mean 
+   and scale of the values. This can enhance visualization and 
+   interpretation of the data
```
Comment (TuomasBorman) [inline inst/pages/11_taxonomic_information.qmd]: standardize transformation also known as Z-transformation --> standardize transformation also known as Z-score

**Comment by TuomasBorman** [comment]
Does this PR take into account the comment here https://github.com/microbiome/OMA/commit/485f8048c01b19aa7398e111076528d3ccfcd872#comments

**Comment by 0xMuluh** [comment]
> Does this PR take into account the comment here [485f804#comments](https://github.com/microbiome/OMA/commit/485f8048c01b19aa7398e111076528d3ccfcd872#comments)

yes.

File: inst/pages/add-comm-typing.Rmd [hunk 7]
```diff
@@ -348,7 +348,7 @@ tse2 <- transformAssay(tse2, MARGIN = "rows", assay.type = "clr", method = "stan
 # get top taxa
 tse2 <- tse2[is.element(rownames(tse2), rel_taxa), ]
 
-mat <- assays(tse2)$z
```
Comment (TuomasBorman) [inline inst/pages/add-comm-typing.Rmd:351]: This does not work. assays() return a list, assay() fetch matrix. 

this should be assay(tse2, "standardize")

File: inst/pages/add-comm-typing.Rmd [hunk 8]
```diff
@@ -514,7 +514,7 @@ names(clust) <- colnames(tse_dmn)
 # get top taxa
 tse_dmn <- tse_dmn[is.element(rownames(tse_dmn), rel_taxa), ]
 # get just counts
-mat <- assays(tse_dmn)$z
```
Comment (TuomasBorman) [inline inst/pages/add-comm-typing.Rmd:517]: Here same

**Review by TuomasBorman — CHANGES_REQUESTED** [review]

File: inst/pages/add-comm-typing.Rmd (hunk 7 above)
Comment (0xMuluh) [reply inst/pages/add-comm-typing.Rmd:351]: hmm I'm not sure I get it.  Is this not the desirable change?
`mat <- assays(tse2, "standardize")`

Comment (TuomasBorman) [reply inst/pages/add-comm-typing.Rmd:351]:
```
tse <- transformAssay(tse, method = "relabundance")

# Does not work
assays(tse, "relabundance")
# Does work
assays(tse)$relabundance
# Does work and preferred
assay(tse, "relabundance")
```

Comment (0xMuluh) [reply inst/pages/add-comm-typing.Rmd:351]: Thanks

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**